python/tests/plotting/test_plot_0d.py

```diff
@@ -3,62 +3,65 @@
 # @file
 # @author Owen Arnold
 
-import scipp as sc
 from scipp import plot
+import scipp as sc
 import pytest
 
 
 def test_plot_0d_variable():
     with pytest.raises(ValueError):
-        plot(sc.Variable(value=1))
+        plot(sc.Variable(value=1)).close()
```

Comment:
We could also make a helper function for the tests, that just does the close, so you don't have to write it everywhere?
```Py
def plot(*args, **kwargs):
    sc.plot(*args, **kwargs).close()
```
and then just have the `plot` call in the tests?